### PR TITLE
feature/CWALL-250

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

**Comment by BtencateSphereon** [comment]
Draft because i need a new artefact, but review can be done

File: packages/web-wallet/pages/contacts/show/index.tsx [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+  const { isLoading, isError, data: partyData } = useOne<Party, HttpError>({
+    resource: 'parties',
+    id
+  })
+
+  const getSupportedCredentials = (): Record<string, CredentialConfigurationSupportedV1_0_13> => { // TODO any
+    try {
+      // TODO returns a map not an array
+      //const supportedCredentials = openID4VCIClient!.getCredentialsSupported() // TODO !
+      const credentialsSupported: Record<string, CredentialConfigurationSupportedV1_0_13> = {
```
Comment (nklomp) [inline packages/web-wallet/pages/contacts/show/index.tsx]: Should get these from the issuer/contact using the oid4vciclient as commented out on the first line. I think we create an array of objects from the record. Which is fine

File: packages/web-wallet/pages/contacts/show/index.tsx [hunk 2]
```diff
@@ -0,0 +1,550 @@
...
+    }
+
+    const credentialIssuer= partyData?.data.identities.find((identity) => identity.roles.includes(CredentialRole.ISSUER))?.identifier.correlationId
+    if (!credentialIssuer) {
+      return
+    }
+
+    OpenID4VCIClient.fromCredentialIssuer({
+      credentialIssuer,
+      authorizationRequest: {
```
Comment (nklomp) [inline packages/web-wallet/pages/contacts/show/index.tsx]: Should not be needed to set. As we will only retrieve the server metadata

File: packages/web-wallet/src/components/views/CredentialCatalogView/index.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+import {CatalogDisplayMode, CredentialCatalogItem, ValueSelection} from "@typings";
+import style from './index.module.css'
+
+type Props = {
+    items: Array<any>
+    onClick?: (item: CredentialCatalogItem) => Promise<void>
+}
+
+const credentialCatalogDisplayModes: Array<ValueSelection> = [
+    {label: 'Card view', value: CatalogDisplayMode.CARD_VIEW},
```
Comment (nklomp) [inline packages/web-wallet/src/components/views/CredentialCatalogView/index.tsx]: i18n

**Review by nklomp — CHANGES_REQUESTED** [review]
See remarks about the current dummy data instead of calling the client and mapping the data

File: packages/web-wallet/pages/contacts/show/index.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply packages/web-wallet/pages/contacts/show/index.tsx]: fixed

File: packages/web-wallet/pages/contacts/show/index.tsx (hunk 2 above)
Comment (BtencateSphereon) [reply packages/web-wallet/pages/contacts/show/index.tsx]: removed

File: packages/web-wallet/src/components/views/CredentialCatalogView/index.tsx (hunk 3 above)
Comment (BtencateSphereon) [reply packages/web-wallet/src/components/views/CredentialCatalogView/index.tsx]: translated

**Review by nklomp — APPROVED** [review]
LGTM

**Pull request merged**